src/HonzaBotner/appsettings.CvutFit.json

```diff
@@ -49,7 +49,7 @@
         ]
     },
     "PinOptions": {
-        "Treshold": 10,
+        "Threshold": 10,
         "TemporaryPinName": ":pushpin:",
         "PermanentPinName": ":round_pushpin:",
         "LockEmojiName": ":lock:",
```

Comment:
Rovnou pod tím v tom RoleToWeightMapping bys mohl přidat ty mody, ať to není na heroku jen lmao